### PR TITLE
Fix 149. Add new regex for password. narek 

### PR DESCRIPTION
**Pull request by narekacy**
Now if user wants to register with password consisting whitespace as first/last symbols, he can't do that. This check is done when user 'Submits' registration form and on the backend

### CODE REVIEW

**Comment by vitaliimak** [comment]
@narekacy could you merge current dev branch into your feature branch?

**Review by vitaliimak — COMMENTED** [review]
Thanks, done yet!

**Review by VozNick — DISMISSED** [review]

**Comment by narekacy** [comment]
My password has more than 8 symbols and strong enough  

image

For better UX positive messages will be better, ("You password should be stronger and contain more than 8 symbols")
Password validation in back-end works well

Answering @vitalii-mak, I've changed this user message and it shows new one now.

I wanted to 'quote answer', but edited previous comment, afterwards wanted to delete my edition, but in fact deleted first comment. Fortunately, our git channel keeps comments history, so I attached this here again.

File: client/src/components/register-login/register.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ class Register extends Component {
       showStrong: false,
       passwordStrength: '',
       passwordInputStarted: false,
-      passwordInputTitle: "Password shouldn't be weak and less than 8 symbols",
+      passwordInputTitle: 'We recommend setting strong password with 8 or more symbols',
```
Comment (vitaliimak) [inline client/src/components/register-login/register.js]: You should explain user when he or she wrongs. This message still doesn't have any information about start or ending with spaces. You can use separate message in that specific case

Comment (vitaliimak) [reply client/src/components/register-login/register.js]: I have 10
image

Comment (narekacy) [reply client/src/components/register-login/register.js]: Done

**Review by vitaliimak — APPROVED** [review]

**Review by VozNick — APPROVED** [review]

**Review by r-yevchuk — APPROVED** [review]

**Pull request merged**